### PR TITLE
Staging next

### PR DESCRIPTION
**Pull request by FRidh**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @

### CODE REVIEW

**Comment by jonringer** [comment]
just curious? what made you back off python38?

major issues i could think of are aiohttp and one other ... I can't remember right now

**Comment by FRidh** [comment]
apparmor still partially fails and with it a lot of other packages, so that's why I reverted it for now

File: pkgs/development/libraries/glog/default.nix [hunk 1]
```diff
@@ -28,7 +28,9 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ cmake ];
 
-  cmakeFlags = [ "-DBUILD_SHARED_LIBS=ON" ];
+  propagatedBuildInputs = [ gflags ];
+
+  cmakeFlags = [ "-DBUILD_SHARED_LIBS=${if static then "OFF" else "ON"}" ];
```
Comment (tobim) [inline pkgs/development/libraries/glog/default.nix]: This merge is not correct, the static option was removed in [4541012](https://github.com/NixOS/nixpkgs/pull/77047/commits/4541012bdbcb5da522e056b75808fa5fa263fc23) which is part of this PR. `cmakeFlags` can be changed back and the `static` option can be removed.

**Review by tobim — CHANGES_REQUESTED** [review]

File: pkgs/development/libraries/glog/default.nix (hunk 1 above)
Comment (FRidh) [reply pkgs/development/libraries/glog/default.nix]: Please open a new PR with the fix.

Comment (tobim) [reply pkgs/development/libraries/glog/default.nix]: #77682

**Pull request merged**